### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
## Checklist

- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/main/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: images/replication-state-diagram.svg [hunk 1]
```diff
@@ -169,7 +169,7 @@
         <path fill="none" stroke="rgb(52,101,164)" d="M 8831,5190 L 12473,5190 13701,5508 12473,5826 8831,5826 7604,5508 8831,5190 8831,5190 Z"/>
         <path fill="none" stroke="rgb(52,101,164)" d="M 7604,5190 L 7604,5190 Z"/>
         <path fill="none" stroke="rgb(52,101,164)" d="M 13701,5826 L 13701,5826 Z"/>
-        <text class="TextShape"><tspan class="TextParagraph" font-family="Liberation Sans, sans-serif" font-size="353px" font-weight="400"><tspan class="TextPosition" x="8613" y="5654"><tspan fill="rgb(0,0,0)" stroke="none">Create job from documen</tspan><tspan font-size="423px" fill="rgb(0,0,0)" stroke="none">t</tspan></tspan></tspan></text>
+        <text class="TextShape"><tspan class="TextParagraph" font-family="Liberation Sans, sans-serif" font-size="353px" font-weight="400"><tspan class="TextPosition" x="8613" y="5654"><tspan fill="rgb(0,0,0)" stroke="none">Create job from document</tspan><tspan font-size="423px" fill="rgb(0,0,0)" stroke="none">t</tspan></tspan></tspan></text>
```
Comment (nickva) [inline images/replication-state-diagram.svg:172]: Good catch. I think in this case though there is a mistaken separate `t` stroke following it. 

We should remove that so we don't have a double `t`

File: src/cluster/nodes.rst [hunk 2]
```diff
@@ -37,7 +37,7 @@ the nodes it is connected to and knows about.
             "node1@xxx.xxx.xxx.xxx"]
     }
 
-* ``all_nodes`` are all the nodes thats this node knows about.
+* ``all_nodes`` are all the nodes that's this node knows about.
```
Comment (flimzy) [inline src/cluster/nodes.rst:40]: This was the wrong correction :)

```suggestion
* ``all_nodes`` are all the nodes that this node knows about.
```

**Review by flimzy — COMMENTED** [review]
One of the corrections isn't actually correct.  Otherwise it looks good, along with a couple other improvements we could make at the same time, or in a future PR.

**Review by nickva — APPROVED** [review]
Looks good and good suggestions from @flimzy as well!

+1

Just needs a rebase on the latest main I think

**Comment by kianmeng** [comment]
> Just needs a rebase on the latest main I think

@nickva done.

**Review by nickva — APPROVED** [review]

**Comment by nickva** [comment]
Merged, thank you again for your contribution, @kianmeng!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**